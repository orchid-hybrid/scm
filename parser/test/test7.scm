(begin (f . g)
       (a x (y . z) . b))
