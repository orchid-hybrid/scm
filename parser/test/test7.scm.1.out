(define-syntax
  when
  (syntax-rules
    ()
    ((when <cond> <body> ...) (if <cond> (begin <body> ...) (values)))))
